packages/eas-cli/src/worker/assets.ts

```diff
@@ -141,6 +141,53 @@ export async function createManifestAsync(
   return { conflictingVariableNames, manifest };
 }
 
+interface RawSourceMap {
+  version: number;
+  sources: string[];
+  names: string[];
+  sourceRoot?: string;
+  sourcesContent?: string[];
+  mappings: string;
+  file: string;
+}
+
+const sourceMapExtensionRe = /\.map$/;
+
+async function normalizeSourceMapAsync(
```

Comment:
Is there any way to add a unit test for this? E.g. https://github.com/cloudflare/workers-sdk/blob/34c71ce9208ffceefe718fc9ae7282ef95e2f2be/packages/wrangler/src/__tests__/deploy.test.ts#L12220

Reply:
There is, but I'll close this PR for now :sweat_smile: this was opened as a test, since I had reasonable confidence this would resolve a source map desymbolication issue. However, it turns out that confidence was unfounded entirely and this didn't fix this issue, which isn't even on our end. That means this PR should be unnecessary for now